.travis.yml

```diff
@@ -14,8 +14,8 @@ after_success:
 
 env:
   global:
-    - secure: "gsbIdYPijtSVfgAeCnAT+qhPGh9vJVsWFLImU0XuvjDV9pNQ+HxL/XQndL0CzIMji+PGdtD1Oeo0Kon/ynJ8wyagk8+VDD1glsBOlr87wdn3sd+LhdxNTFM4XYmw3Wq6i5Hs+HwtoixWRLS64dsEvNrf07XYQN0VbdLmqhytCcs="
-    - secure: "g7JbR/gRci59QplZKq5Xh85XMbdcBD97knKEG46Yiyfay3K+WThIPHer5FPOV65lgM5c7zw2uTAqpOFhLUtzr1dbGZwhLrpkGX5r07UmV4ByoOMaV5Cb34Lay53BqZe7AHCtn66csaZMTJsQvGD/7XrE+UuX5sktI3RffFPhtic="
+    - secure: "TODO" # recreate as ORG_GRADLE_PROJECT_SONATYPE_NEXUS_USERNAME env var
+    - secure: "TODO" # recreate as ORG_GRADLE_PROJECT_SONATYPE_NEXUS_PASSWORD env var
```

Comment:
This is for you to do. If you want to generate them and comment I can add to this commit.